pkg/monitortests/node/kubeletlogcollector/monitortest.go

```diff
@@ -60,3 +66,30 @@ func (*kubeletLogCollector) Cleanup(ctx context.Context) error {
 	// TODO wire up the start to a context we can kill here
 	return nil
 }
+
+func nodeFailedLeaseErrors(finalIntervals monitorapi.Intervals) []*junitapi.JUnitTestCase {
+	const testName = "[sig-node] kubelet-log-collector detects node failed to lease events"
```

Comment:
Drive by. I wish the client-go based components could produce the audit logs instead of parsing a log file to find failed requests.